app/models/location.py

```diff
@@ -35,14 +44,14 @@ def create_zip_code(zip_code):
         return result
 
     @staticmethod
-    def generate_fake(count=10):
-        """Generate count fake ZIPCodes for testing."""
-        from faker import Faker
+    def generate_fake():
+        """Use a list of arbitrary but real zipcodes because the faker library
```

Comment:
`"""Populate the zip_codes table with arbitrary but real zip codes. The zip codes generated by the faker library are not necessarily valid or guaranteed to be located in the US."""`